CheckDrive.Api/CheckDrive.Api/Controllers/Authorization/AuthController.cs

```diff
@@ -0,0 +1,43 @@
+﻿using CheckDrive.Application.Constants;
+using CheckDrive.Application.DTOs.Identity;
+using CheckDrive.Application.Interfaces.Authorization;
+using Microsoft.AspNetCore.Authorization;
+using Microsoft.AspNetCore.Mvc;
+
+namespace CheckDrive.Api.Controllers.Authorization;
+
+[Route("api/auth")]
+[ApiController]
+public class AuthController : ControllerBase
+{
+    private readonly IAuthService _authService;
+
+    public AuthController(IAuthService authService)
+    {
+        _authService = authService
+            ?? throw new ArgumentNullException(nameof(authService));
+    }
+
+    [HttpPost("register/employee")]
+    [Authorize(Roles = $"{Roles.Manager},{Roles.Administrator}")]
+    public async Task<IActionResult> Register(RegisterDto registerUser)
+    {
+        await _authService.RegisterEmployeeAsync(registerUser);
+        return Created();
```

Comment:
https://stackoverflow.com/questions/37839278/asp-net-core-rc2-web-api-post-when-to-use-create-createdataction-vs-created